### PR TITLE
raidboss: p8s initial vent bugfix

### PR DESCRIPTION
**Pull request by quisquous**
Sometimes the combatants are added after the startsCasting line, and so look for their positions as soon as they are added.  Additionally, some of these combatants are added at the start of the fight, so ignore them until we see Abyssal Fires.

### CODE REVIEW

**Comment by Legends0** [comment]
Ah, I can try this.

File: ui/raidboss/data/06-ew/raid/p8s.ts [hunk 1]
```diff
@@ -869,24 +868,33 @@ const triggerSet: TriggerSet<Data> = {
...
+          data.ventIds.push(matches.id);
+      },
       promise: async (data: Data) => {
         data.combatantData = [];
-        if (data.ventIds.length !== 2)
+        const ventIds = data.ventIds;
+        if (ventIds === undefined)
+          return;
+
+        if (ventIds.length !== 2)
```
Comment (panicstevenson) [inline ui/raidboss/data/06-ew/raid/p8s.ts:894]: nit:
```suggestion
        if (ventIds?.length !== 2)
```

**Review by panicstevenson — APPROVED** [review]

**Comment by Legends0** [comment]
This needs a conditional block... otherwise it outputs on loading of the zone/stage reset.

**Comment by Legends0** [comment]
I think this is just calling the first vent...

**Comment by quisquous** [comment]
Going to close this in favor of #4830.

**Pull request closed without merging**